azure-pipelines.yml

```diff
@@ -80,6 +81,9 @@ jobs:
     condition: and(succeeded(), eq(variables['BUILD_TYPE'], 'wheel'))
   - script: python -m pip install --upgrade pip setuptools wheel
     displayName: 'Install tools'
+  - script: python -m pip install $(PIP_PRE_EXTRAS)
+    displayName: 'Pre-deps extras'
+    condition: ne(variables['PIP_PRE_EXTRAS'], '')
```

Comment:
Won't this get overridden by the `pip install scipy` below? I suppose another approach might be to use `--only-binary="h5py,scipy"` or whatever the exact syntax for that is, to avoid trying a source install of `1.9.2`.

Apart from that, your assessment is consistent with my understanding--SciPy and scitkit-learn seem to be on a path to removing win 32-bit support, though some debate still seems to be happening re: some folks using 32-bit Python on Windows if only by some default Windows download mechansim.

Reply:
ah, I guess there's no `upgrade` flag so you're ok

Reply:
o yeah that's very true only binary might be cleaner, I'm happy to go with that if you like that option too

Reply:
too late, o well there's a couple of uses to that extra step I can think of anyways